### ISSUE TITLE
BatchNormalization

### ISSUE THREAD

**Issue by niesocja** [tensorflow/tensorflow#51899]
I am trying to understand how BatchNormalization works. With code:
`x=tf.constant([2.,3.,4.])`
`x=keras.layers.BatchNormalization()(x)`
`print(x)`
I get response: 
`tf.Tensor([1.9990008 2.9985013 3.9980016], shape=(3,), dtype=float32)`
Which is not what i expect since it doesn't have mean 0 or variance 1. Can somebody explain?

**Comment by tilakrayal** [comment]
@niesocja ,
Please refer this links [1](https://www.analyticsvidhya.com/blog/2021/03/introduction-to-batch-normalization/) and [2](https://www.tensorflow.org/api_docs/python/tf/keras/layers/BatchNormalization) which provide the information on BatchNormalization.It helps.Thanks!
